jsonnet/rules.libsonnet

```
@@ -381,6 +381,48 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
               severity: 'info',
             },
           },
+          {
+            alert: 'KubePersistentVolumeFillingUp',
+            expr: |||
+              kubelet_volume_stats_used_bytes{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kubelet", metrics_path="/metrics"} > 0
+              and
+              (
+                kubelet_volume_stats_available_bytes{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kubelet", metrics_path="/metrics"}
+                    /
+                kubelet_volume_stats_capacity_bytes{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kubelet", metrics_path="/metrics"}
+              ) < 0.03
```

The returned value is the first part of the expression before the `and` so the `kubelet_volume_stats_used_bytes... > 0` should go in second position.

```suggestion
              (
                kubelet_volume_stats_available_bytes{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kubelet", metrics_path="/metrics"}
                    /
                kubelet_volume_stats_capacity_bytes{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kubelet", metrics_path="/metrics"}
              ) < 0.03
              and
              kubelet_volume_stats_used_bytes{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kubelet", metrics_path="/metrics"} > 0
```

---

I will fix it. Thanks